style.css

```diff
@@ -0,0 +1,185 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  width: 375px !important;
+  height: 6715px;
+  text-align: center;
+  justify-content: space-around;
+  background-color: #e5e5e5;
+}
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-menu {
+  display: flex;
+  gap: 30px;
+}
+
+.nav-title {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 28px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: #3a4a42;
+}
+
+.handburger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  background-color: #172b4d;
+  margin: 5px auto;
+  -webkit-transition: all 0.3 ease-in-out;
+  transition: all 0.3 ease-in-out;
+}
+
+.first-section {
+  background-image: url("./Images//mobile-bg.svg");
+  background-repeat: no-repeat;
+  height: 648px;
+  width: 100%;
+}
```

Comment:
Good job on the project so far 👍🏽 the styling is almost perfect 🎯 but this would be better using `flexbox`, by default the display of the section is `block` but `flex` gives us a lot of options to align all our elements, plus, it's responsive. 💯 

You'd of course need to adjust the `flex-direction` as well because I think it's `row` by default. It could look like this:
```
.first-section {
  display: flex;
  flex-direction: column;
  background-image: url("./Images//mobile-bg.svg");
  background-repeat: no-repeat;
  height: 648px;
  width: 100%;
}
```